internal/pkg/api/server/templates/service/service.go

```diff
@@ -201,6 +201,7 @@ func getTemplateVersion(d dependencies.Container, repoName, templateId, versionS
 				Message: fmt.Sprintf(`Version "%s" not found.`, versionStr),
 			}
 		}
+		return nil, err
```

Comment:
Ak chyba nezodpoveda ziadne z tych vyssie, tak je to interna chyba, ktoru je potrebne vratit.

Reply:
Nevalidnu sablonu budeme testovat vramci E2E testov, ked ich spravime.